api_employees.go

```diff
@@ -197,7 +197,7 @@ func (r ApiEmployeesIgnoreCreateRequest) IgnoreCommonModelRequest(ignoreCommonMo
 	return r
 }
 
-func (r ApiEmployeesIgnoreCreateRequest) Execute() (IgnoreCommonModel, *_nethttp.Response, error) {
+func (r ApiEmployeesIgnoreCreateRequest) Execute() (*_nethttp.Response, error) {
```

Comment:
Based on https://docs.merge.dev/hris/employees/#employees_ignore_create I'm surprised this parameter went away, is it legit?

Reply:
Oh i see, silly Go, the input didnt change it's the output?